### ISSUE TITLE
Extracting softmax scores on prediction

### ISSUE THREAD

**Issue by AndersGiovanni** [ebanalyse/NERDA#32]
Hi @smaakage85,

Thank you very much for your work on the NERDA project, very much appreciated! 

In the `predict()` function found in [predictions.py](https://github.com/ebanalyse/NERDA/blob/ae45d7e5368059721d1073384201433ea7a6e820/src/NERDA/predictions.py#L15) you've added the functionality to return `return_confidence` which is super nice! However, when you extract the most likely tag using `values, indices = outputs[i].max(dim=1)` (from line 104) the prediction scores have not been put in a [softmax function](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html) which would give confidence scores of each tag relative to the others.

It would be very nice to either
1. Being able to get all the tag probabilities for each prediction.
2. Apply a softmax function in either `predict()` or in `forward()` ([NERDANetwork](https://github.com/ebanalyse/NERDA/blob/ae45d7e5368059721d1073384201433ea7a6e820/src/NERDA/networks.py#L7)) like they do in [this basic pytorch tutorial](https://pytorch.org/tutorials/beginner/nlp/deep_learning_tutorial.html).
